EIPS/eip-5560.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: 5560
+title: Redeemable NFTs
+description: Links physical objects to NFTs
+author: Olivier Fernandez (@fernandezOli), Frédéric Le Coidic (@FredLC29), Julien Béranger (@julienbrg)
+discussions-to: https://ethereum-magicians.org/t/eip-redeemable-nft-extension/10589
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-30
+requires: 165, 721
+---
+
+## Abstract
+
+The Redeemable NFT Extension adds a `redeem` function to [EIP-721](./eip-721.md). It can be implemented when an NFT issuer wants his/her NFT to be redeemed for a physical object.
+
+## Motivation
+
+An increasing amount of NFT issuers such as artists, fine art galeries, auction houses, brands and others want to offer a physical object to the holder of a given NFT. This standard allows EIP-721 NFTs to signal reedemability.
+
+## Specification
+
+_The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119._
+
+**EIP-721 compliant contracts MAY implement this EIP to provide a standard method of receiving information on redeemability.**
+
+The NFT issuer **MUST** decide who is allowed to redeem the NFT, and restrict access to the `redeem()` function accordingly.
+
+Anyone **MAY** access the `isRedeemable()` function to check the redeemability status: it returns `true` when the NFT redeemable, and `false` when already redeemed.
+
+Third-party services that support this standard **MAY** use the `Redeem` event to listen to changes on the redeemable status of the NFT.
+
+Implementers of this standard **MUST** have all of the following functions:
+
+```solidity
+import '@openzeppelin/contracts/utils/introspection/ERC165.sol';
+
+/**
+ * @dev Implementation of Redeemable for ERC-721s
+ *
+ */
+
+interface IRedeemable is ERC165 {
```

Comment:
[Technical, peer-review, optional] Consider add a "event" for "Redeemed" when a redemption occurs?

Reply:
Good point. Thanks again, we added it in the interface.